Lib/bisect.py

```diff
@@ -17,7 +17,7 @@ def insort_right(a, x, lo=0, hi=None):
         mid = (lo+hi)//2
         if x < a[mid]: hi = mid
         else: lo = mid+1
-    a.insert(lo, x)
+    a[lo:lo] = x
```

Comment:
They aren't the same though. `a.insert(lo, x)` inserts an item into `a`. `a[lo:lo]=x` inserts all the items of an iterable into `a` (and fails if `x` isn't iterable).

Did you test this?

Reply:
ok~ seems reasonable